modules/widgets/facebook-likebox.php

```diff
@@ -76,9 +76,11 @@ function widget( $args, $instance ) {
 		$title    = apply_filters( 'widget_title', $instance['title'] );
 		$page_url = set_url_scheme( $like_args['href'], 'https' );
 
-		$like_args['show_faces'] = (bool) $like_args['show_faces'] ? 'true' : 'false';
-		$like_args['stream']     = (bool) $like_args['stream'] ? 'true' : 'false';
-		$like_args['cover']      = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['show_faces'] 		= (bool) $like_args['show_faces'] ? 'true' : 'false';
```

Comment:
You seem to have a mix of spaces and tabs here. Ideally you'd only have spaces to indent this.

Same below.

Reference: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation